components/schematics/src/ng-update/tests/root-imports.spec.ts

```diff
@@ -65,4 +65,33 @@ describe('v5 dynatrace angular components imports', () => {
 
     removeTempDir();
   });
+
+  it('should migrate root imports in .spec files', async () => {
+    const {
+      runFixers,
+      appTree,
+      writeFile,
+      removeTempDir,
+    } = await createTestCaseSetup('update-5.0.0', migrationCollection, []);
+
+    writeFile(
+      'projects/lib-testing/src/main.spec.ts',
```

Comment:
please check if the tsconfig inside the testing case setup is configured to work with the spec files otherwise the ts program that will be created cannot work.

Please check how material is testing this otherwise it is kind of useless.